mod_upload/controllers.py

```diff
@@ -308,6 +308,7 @@ def process_id(upload_id):
                                 '{notes}\n*Description:*\n' \
                                 '{desc}'.format(notes=form.notes.data,
                                                 desc=form.IssueBody.data)
+                        # Make the issue on GitHub issue tracker
```

Comment:
The line below ("make_github_issue") is already descriptive enough. I don't see the need for an additional comment here.